addon/templates/components/polaris-drop-zone/file-upload.hbs

```diff
@@ -13,11 +13,11 @@
 {{else if (eq size "large")}}
   {{#polaris-stack vertical=true spacing="tight"}}
     {{#if (eq type "file")}}
-      {{svg-jar assetFileUpload class=imageClasses}}
+      {{svg-jar fileUpload class=imageClasses}}
```

Comment:
this might be my bad when initially implemented....but instead of an `svg` elements, we should be using `img`

Reply:
don't think there was a specific reason for doing this